vipps-report-api-faq.md

```diff
@@ -33,6 +34,47 @@ We are still quite early in the planning and design phase and will use the
 draft documentation here to discuss with potential users of the API
 and make sure we are developing the right solution.
 
+## Which API keys give access to the API?
+
+In short: These API keys give access to the Report API:
+* The merchant's own API keys (the same API keys that are used to make payments)
+* Partner keys (the same API keys that are used to make payments)
+* Accounting partners' API keys (after the merchant has given the accounting partner access)
+
+### The merchant's own API keys
+
+The merchant's own API keys give full access to the Report API.
+
+If the merchant uses an integration partner (see
+[partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/#partner-types)),
+it is the same as using the merchant's own API keys.
+
+When a merchant shares it's API keys for a sale unit with an integration partner,
+Vipps has no way of knowing whether the API calls are made by the merchant or
+the integration partner.
```

Comment:
When a merchant shares it's API keys for a sale unit with an integration partner,
Vipps has no way of knowing whether the API calls are made by the merchant or
the integration partner.*
---
How about rephrasing to:

If a merchant shares it's API keys for a sale unit with an integration partner,
Vipps has no way of knowing whether the API calls are made by the merchant or
the integration partner. This is why we mandate that all requests made by a partner on behalf of a Merchant is done by using the Partner's own keys.

Reply:
Good point. Added:
>This is why we require that all API requests made by a partner on behalf of a
merchant are done using the partner's own API keys (the partner keys).